driver.go

```diff
@@ -220,6 +215,14 @@ func (l *LinstorDriver) Create(req *volume.CreateRequest) error {
 	if err != nil {
 		return err
 	}
+	defer func() {
```

Comment:
hm, personally I'm really not a fan of deferred functions that rely on global state (i.e., `err`), it takes a lot of care to (not) mess with that state in the rest of the function/or mess with it at the right time.

I would suggest to wrap the function. Have a "low level" create which returns an `error` and the "top level" function that calls the low level, checks, and rolls back if necessary.